widget.subscribe.php

```diff
@@ -61,24 +61,23 @@ public function __construct() {
 			)
 		);
 
-		// Only register widget scripts, styles, and ajax when widget is active
-		if ( is_active_widget( false, false, $this->id_base, true ) ) {
+		if ( is_active_widget( false, false, $this->id_base, true ) || shortcode_exists( 'sailthru_widget' ) ) {
+			// Only register widget scripts, styles, and ajax when widget is active
 			// Register admin styles and scripts
 			// According to documentation: admin_print_styles should not be used to enqueue styles or scripts on the admin pages. Use admin_enqueue_scripts instead.
 			add_action( 'admin_enqueue_scripts', array( $this, 'register_admin_scripts' ) );
 
 			// Register site styles and scripts
-			add_action( 'wp_enqueue_scripts', array( $this, 'register_widget_styles' ) );
-			add_action( 'wp_enqueue_scripts', array( $this, 'register_widget_scripts' ) );
 
 			// Include the Ajax library on the front end
-			add_action( 'wp_head', array( $this, 'add_ajax_library' ) );
 
 			// Method to subscribe a user
 			add_action( 'wp_ajax_nopriv_add_subscriber', array( $this, 'add_subscriber' ) );
 			add_action( 'wp_ajax_add_subscriber', array( $this, 'add_subscriber' ) );
+			add_action( 'wp_enqueue_scripts', array( $this, 'register_widget_styles' ) );
+			add_action( 'wp_enqueue_scripts', array( $this, 'register_widget_scripts' ) );
+			add_action( 'wp_head', array( $this, 'add_ajax_library' ) );
```

Comment:
These comments can be removed above?
`// Register site styles and scripts` 
`// Include the Ajax library on the front end`